phaser/game.js

```diff
@@ -46,10 +50,16 @@ function preload (){
   this.load.spritesheet('ship', 'assets/ship.png', { frameWidth: 90, frameHeight: 90 })
   this.load.image('laserGreen', 'assets/laserGreenR.png')
   this.load.image('laserBlue', 'assets/laserBlueR.png')
+  this.load.image('shield1', 'assets/shield1.png')
+  this.load.image('shield2', 'assets/shield2.png')
+  this.load.image('shield_powerup', 'assets/shield_gold.png')
+  this.load.image('gold_powerup', 'assets/bolt_gold.png')
+  this.load.image('silver_powerup', 'assets/bolt_silver.png')
 }
 
 function create (){
   let self = this;
+  scene = this;
```

Comment:
What are you using scene for?

Reply:
To have it globally defined. I think there's a better way to access the scene through one of the game objects but I couldn't remember how to do it. I'll look into it again.